.github/workflows/provision-minikube.yml

```diff
@@ -55,15 +55,88 @@ jobs:
           github token: ${{ secrets.GITHUB_TOKEN }}
           driver: docker
           start args: --memory 4096
-      - name: Provision everything
+      - name: Provision Legacy
         working-directory: provision/minikube
         run: |
           PATH=$PATH:$GITHUB_WORKSPACE/bin
           ./rebuild.sh
           task # default configuration PostgreSQL and legacy store
+          kubectl get pods -A
+          kubectl get events -A
+      - name: Create Dataset on Postgres Legacy Store
+        working-directory: dataset
+        run: |
+          ./dataset-import.sh --clear-status-completed
+          ./dataset-import.sh -r '1 5 5 1000'
+          ./dataset-import.sh --status-completed
+          EXIT_CODE=$?
+          test $EXIT_CODE -eq 0 && echo "Creation of realm successful" || echo "something bad happened"
+          exit $EXIT_CODE
```

Comment:
This code is not necessary as this is what GitHub actions will do anyway: once a command returns not zero, the step will fail.

Reply:
ok gtk, i will remove this line of code, then. thank you @ahus1

Reply:
removed this in the latest commit